src/cookbooks/bigquery/querying.md

```diff
@@ -154,7 +154,7 @@ You can create views in BigQuery if you have access via [GCP BigQuery Console](a
 
 BigQuery offers [user-defined functions](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions) (UDFs) that can be defined in SQL or JavaScript as part of a query or as a persistent function stored in a dataset. We have defined a suite of persistent functions to enable transformations specific to our data formats, available in datasets `udf` (for functions defined in SQL) and `udf_js` (for functions defined in JavaScript). Note that JavaScript functions are potentially much slower than those defined in SQL, so use functions in `udf_js` with some caution, likely only after performing aggregation in your query.
 
-We document a few of the most broadly useful UDFs below, but you can see the full list of UDFs with source code in [`bigquery-etl/udf`](https://github.com/mozilla/bigquery-etl/tree/master/udf) and [`bigquery-etl/udf_js`](https://github.com/mozilla/bigquery-etl/tree/master/udf_js). Publishing a full reference page for our persistent UDFs is a planned improvement, tracked in [`bigquery-etl#228`](https://github.com/mozilla/bigquery-etl/issues/228).
+We document a few of the most broadly useful UDFs below, but you can see the full list of UDFs with source code in [`bigquery-etl/sql/moz-fx-data-shared-prod/udf`](https://github.com/mozilla/bigquery-etl/tree/master/sql/moz-fx-data-shared-prod/udf) and [`bigquery-etl/sql/moz-fx-data-shared-prod/udf_js`](https://github.com/mozilla/bigquery-etl/tree/master/sql/moz-fx-data-shared-prod/udf_js). Publishing a full reference page for our persistent UDFs is a planned improvement, tracked in [`bigquery-etl#228`](https://github.com/mozilla/bigquery-etl/issues/228).
```

Comment:
This section should mention `mozfun` and its published docs now.

Reply:
I just noticed bigquery-etl#228, it should be pretty easy to adopt the docs publishing and also publish docs shared-prod UDFs the same way we publish them for mozfun once the alternate project support has landed